ts/features/itwallet/issuance/screens/ItwIssuanceEidPreviewScreen.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  ButtonLink,
+  ButtonSolid,
+  ContentWrapper,
+  IOColors,
+  VSpacer,
+  useIOTheme
+} from "@pagopa/io-app-design-system";
+import * as O from "fp-ts/lib/Option";
+import { pipe } from "fp-ts/lib/function";
+import React from "react";
+import { ColorValue, StyleSheet, View } from "react-native";
+import { OperationResultScreenContent } from "../../../../components/screens/OperationResultScreenContent";
+import { IOScrollViewWithLargeHeader } from "../../../../components/ui/IOScrollViewWithLargeHeader";
+import I18n from "../../../../i18n";
+import { useIONavigation } from "../../../../navigation/params/AppParamsList";
+import { EidCardPreview } from "../../common/components/EidCardPreview";
+import { ItwCredentialClaimsList } from "../../common/components/ItwCredentialClaimList";
+import {
+  ItWalletError,
+  getItwGenericMappedError
+} from "../../common/utils/itwErrorsUtils";
+import { ItwCredentialsMocks } from "../../common/utils/itwMocksUtils";
+import { StoredCredential } from "../../common/utils/itwTypesUtils";
+
+export const ItwIssuanceEidPreviewScreen = () => {
+  const navigation = useIONavigation();
+  const eidOption = O.some(ItwCredentialsMocks.eid);
+
+  /**
+   * Renders the content of the screen if the PID is decoded.
+   * @param eid - the decoded eID
+   */
+  const ContentView = ({ eid }: { eid: StoredCredential }) => {
+    const theme = useIOTheme();
+
+    const backgroundColor: ColorValue =
+      IOColors[theme["appBackground-primary"]];
+
+    React.useLayoutEffect(() => {
+      navigation.setOptions({
+        headerShown: true
+      });
+    }, []);
+
+    return (
+      <IOScrollViewWithLargeHeader
+        title={{
+          label: I18n.t("features.itWallet.issuance.credentialPreview.title", {
+            credential: eid.displayData.title
+          })
+        }}
+      >
+        <View style={styles.preview}>
+          <EidCardPreview />
+        </View>
+        <View style={styles.dropShadow}>
+          <VSpacer size={24} />
+        </View>
+        <View style={[styles.content, { backgroundColor }]}>
+          <ItwCredentialClaimsList data={eid} />
+        </View>
+        <ContentWrapper>
+          <VSpacer size={32} />
+          <ButtonSolid
+            label={I18n.t(
+              "features.itWallet.issuance.credentialPreview.actions.primary"
+            )}
+            icon="add"
+            iconPosition="end"
+            onPress={() => undefined}
+            fullWidth={true}
+          />
+          <VSpacer size={24} />
+          <View style={{ alignSelf: "center" }}>
+            <ButtonLink
+              label={I18n.t(
+                "features.itWallet.issuance.credentialPreview.actions.secondary"
+              )}
+              onPress={() => undefined}
+            />
+          </View>
+        </ContentWrapper>
```

Comment:
Just replaced this chunk of code but as @mastro993 pointed out, an extra margin was applied at the end of the page. I've already changed `IOScrollView` and `IOScrollViewWithLargeHeader` with an extra prop called `excludeEndContentMargin`. Let me know if I can add the change to this PR or if you prefer to manage it in a different PR
```suggestion
<FooterActions
          fixed={false}
          actions={{
            type: "TwoButtons",
            primary: {
              icon: "add",
              iconPosition: "end",
              label: I18n.t(
                "features.itWallet.issuance.credentialPreview.actions.primary"
              ),
              onPress: () => undefined
            },
            secondary: {
              label: I18n.t(
                "features.itWallet.issuance.credentialPreview.actions.secondary"
              ),
              onPress: () => undefined
            }
          }}
        />
```

Reply:
> Just replaced this chunk of code but as @mastro993 pointed out, an extra margin was applied at the end of the page. I've already changed `IOScrollView` and `IOScrollViewWithLargeHeader` with an extra prop called `excludeEndContentMargin`. Let me know if I can add the change to this PR or if you prefer to manage it in a different PR

If we already have the solution, let's add it to this PR! 🚀

Reply:
@mastro993 Addressed in the following commit → https://github.com/pagopa/io-app/pull/5780/commits/874a314daca5c6755f7eb230bed66ff73df0984b. Let me know if everything is okay ✌🏻 

(Tested on iPhone SE, first generation one, and here's how that turned out)

<img src="https://github.com/pagopa/io-app/assets/1255491/8209240a-6dec-4e91-be80-181c14734a8f" width="320" />

Reply:
It works as expected. Thank you!